L1Trigger/Phase2L1ParticleFlow/plugins/L1TPFProducer.cc

```diff
@@ -144,12 +154,27 @@ L1TPFProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
     }
 
     /// ------ READ MUONS ----
-    edm::Handle<l1t::MuonBxCollection> muons;
-    iEvent.getByToken(muCands_, muons);
-    for (auto it = muons->begin(0), ed = muons->end(0); it != ed; ++it) {
-        const l1t::Muon & mu = *it;
-        if (debugR_ > 0 && deltaR(mu.eta(),mu.phi(),debugEta_,debugPhi_) > debugR_) continue;
-        l1regions_.addMuon(mu, l1t::PFCandidate::MuonRef(muons, muons->key(it)));
+    if(useStandaloneMuons_ && !useTrackerMuons_) {
```

Comment:
rather than the `&&!useTrackerMuons_` here, I would put in the constructor a check that one is not enabling both
```
if (useStandaloneMuons_ && useTrackerMuons_) {
     throw cms::Exception("Configuration", "setting useStandaloneMuons=True && useTrackerMuons=True is not to be done, as it would duplicate all muons\n");
}
````